client/src/helpers/history.helper.ts

```diff
@@ -1,3 +1,3 @@
 import { createBrowserHistory } from 'history';
 
-export default createBrowserHistory();
+export default createBrowserHistory();
```

Comment:
Add newline after createBrowserHistory(); (end-of-source line)

Reply:
done